torchsynth/module.py

```diff
@@ -613,11 +689,13 @@ def _forward(self, mod_signal: Signal) -> Signal:
         """
         Must be implemented in deriving classes
         """
+        # This module accepts signals at control rate!
```

Comment:
hmmm yeah okay maybe SignalModule is the wrong word, since we really mean ControlRateModule vs AudioModule or something